sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdEndpointStatistics.java

```diff
@@ -83,6 +88,10 @@ public void serialize(RntbdEndpointStatistics stats,
             writer.writeStringField("lastRequestTime", toInstantString(stats.lastRequestNanoTime));
             writer.writeStringField("createdTime", toInstantString(stats.createdTime));
             writer.writeBooleanField("isClosed", stats.closed);
+            if (stats.connectionStateListenerMetrics != null)
+            {
+                writer.writeObjectField("cerMetrics", stats.connectionStateListenerMetrics);
```

Comment:
why is this called `cerMetrics`? Can we change this to a better name? Hard to read as cerMetrics ?